spring-integration-core/src/main/java/org/springframework/integration/dsl/context/StandardIntegrationFlowContext.java

```diff
@@ -44,9 +44,7 @@
  *
  * @author Artem Bilan
  * @author Gary Russell
- *
```

Comment:
Would you mind, please, to revert all the code style changes?

This blank line really has to be here to divide `@author` list from `@since`.

Also add you name, please, to this class.